src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseDetailsEntity.java

```diff
@@ -95,6 +96,9 @@
     @Column(name = DATA_CLASSIFICATION_COL, nullable = false)
     @Convert(converter = JsonDataConverter.class)
     private JsonNode dataClassification;
+    @Column(name = SUPPLEMENTARY_DATA_COL, nullable = false)
```

Comment:
in theory the column is nullable: 
https://github.com/hmcts/ccd-data-store-api/blob/95a9061f27d9d059fab03aaba5b20704fe20b58e/src/main/resources/db/changelog/db.changelog-RDM-8747.xml#L8